[2025-02-23T15:52:45.942+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-23T15:52:45.964+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-23T15:52:45.976+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-23T15:52:45.977+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-23T15:52:45.993+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-18 00:00:00+00:00
[2025-02-23T15:52:46.001+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1592) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-23T15:52:46.003+0000] {standard_task_runner.py:72} INFO - Started process 1599 to run task
[2025-02-23T15:52:46.002+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-18T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmphrp3dbt4']
[2025-02-23T15:52:46.005+0000] {standard_task_runner.py:105} INFO - Job 64: Subtask third_task
[2025-02-23T15:52:46.058+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-18T00:00:00+00:00 [running]> on host 2cebbf768e94
[2025-02-23T15:52:46.153+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:00:00+00:00'
[2025-02-23T15:52:46.154+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-23T15:52:46.170+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-23T15:52:46.170+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-23T15:52:46.179+0000] {subprocess.py:99} INFO - Output:
[2025-02-23T15:52:46.180+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-23T15:52:46.180+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-23T15:52:46.202+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-23T15:52:46.203+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-18T00:00:00+00:00, execution_date=20250218T000000, start_date=20250223T155245, end_date=20250223T155246
[2025-02-23T15:52:46.261+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-23T15:52:46.276+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-23T15:52:46.278+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-24T04:47:22.363+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-24T04:47:22.395+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-24T04:47:22.416+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.third_task scheduled__2025-02-18T00:00:00+00:00 [queued]>
[2025-02-24T04:47:22.417+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-02-24T04:47:22.441+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): third_task> on 2025-02-18 00:00:00+00:00
[2025-02-24T04:47:22.450+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1116) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-24T04:47:22.452+0000] {standard_task_runner.py:72} INFO - Started process 1126 to run task
[2025-02-24T04:47:22.451+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'third_task', 'scheduled__2025-02-18T00:00:00+00:00', '--job-id', '92', '--raw', '--subdir', 'DAGS_FOLDER/my_first_dags.py', '--cfg-path', '/tmp/tmpsyxzykzg']
[2025-02-24T04:47:22.453+0000] {standard_task_runner.py:105} INFO - Job 92: Subtask third_task
[2025-02-24T04:47:22.525+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.third_task scheduled__2025-02-18T00:00:00+00:00 [running]> on host 79e0cb3ed452
[2025-02-24T04:47:22.660+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='gzang' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2025-02-18T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-18T00:00:00+00:00'
[2025-02-24T04:47:22.663+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-24T04:47:22.687+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-02-24T04:47:22.689+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo finally, it will be done!!!']
[2025-02-24T04:47:22.713+0000] {subprocess.py:99} INFO - Output:
[2025-02-24T04:47:22.714+0000] {subprocess.py:106} INFO - finally, it will be done!!!
[2025-02-24T04:47:22.715+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-02-24T04:47:22.764+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-24T04:47:22.767+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=third_task, run_id=scheduled__2025-02-18T00:00:00+00:00, execution_date=20250218T000000, start_date=20250224T044722, end_date=20250224T044722
[2025-02-24T04:47:22.875+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-24T04:47:22.925+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-24T04:47:22.929+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
